example/ios/Podfile

```diff
@@ -71,9 +71,9 @@ target 'ReactNativeSample' do
   #
   # Note that if you have use_frameworks! enabled, Flipper will not work and
   # you should disable these next few lines.
-  add_flipper_pods!
+  # add_flipper_pods!
```

Comment:
I ended up commenting flipper out. 
It looks like it's not even recommended anymore in the most recent version. 

I tried upgrading, but auto-upgrade totally breaks. I think we should eventually kill this example altogether and start a new one, so that we can start with the latest RN version. we can just port our files over. 
https://react-native-community.github.io/upgrade-helper/?from=0.62.2&to=0.64.0